addon/components/sl-split-grid.js

```diff
@@ -0,0 +1,564 @@
+     *
+     * @function actionsColumnStyle
+     * @observes actionsColumnWidth
+     * @returns  {string} - The `style` attribute for the column
+     */
+    actionsColumnStyle: function() {
+        return 'width: ' + this.get( 'actionsColumnWidth' ) + 'px;';
+    }.property( 'actionsColumnWidth' ),
+
+    /**
+     * The desired title for the detail pane, based on `detailTitlePath`
+     *
+     * @function detailTitle
+     * @observes activeRecord, detailTitlePath
+     * @returns  {void}
+     */
+    detailTitle: function() {
+        var activeRecord = this.get( 'activeRecord' );
+
+        if ( activeRecord ) {
+            return Ember.get( activeRecord, this.get( 'detailTitlePath' ) );
+        }
+    }.property( 'activeRecord', 'detailTitlePath' ),
+
+    /**
+     * Disables the scroll event handling for continuous paging
+     *
+     * @function disableContinuousPaging
+     * @returns  {void}
+     */
+    disableContinuousPaging: function() {
+        this.$( '.list-pane .content' ).unbind( 'scroll' );
+    },
+
+    /**
+     * Enables the scroll event handling for continuous paging
+     *
+     * @function enableContinuousPaging
+     * @returns  {void}
+     */
+    enableContinuousPaging: function() {
+        this.$( '.list-pane .content' ).bind( 'scroll', this.handleListContentScroll.bind( this ) );
+    },
+
+    /**
+     * Retrieve the sorted column definition
+     *
+     * @function getSortedColumn
+     * @returns  {object} - The definition for the currently sorted column
+     */
+    getSortedColumn: function() {
+        var columns           = this.get( 'columns' ),
+            sortedColumnTitle = this.get( 'sortedColumnTitle' );
+
+        if ( sortedColumnTitle ) {
+            for ( var i = 0; i < columns.length; i++ ) {
+                if ( Ember.get( columns[ i ], 'title' ) === sortedColumnTitle ) {
+                    return columns[ i ];
+                }
+            }
+        }
+    },
+
+    /**
+     * Callback to the list content scrolling, which is responsible for
+     * determining when triggering nextPage is necessary by checking the
+     * scroll location of the content
+     *
+     * @function handleListContentScroll
+     * @param    {jQuery.Event} event - The scroll trigger event
+     * @returns  {void}
+     */
+    handleListContentScroll: function( event ) {
+        var listContent  = this.$( event.target ),
+            scrollBottom = listContent.scrollTop() + listContent.height();
+
+        if ( scrollBottom >= this.get( 'nextPageScrollPoint' ) && !this.get( 'pendingData' ) ) {
+            this.requestNextPage();
+        }
+    },
+
+    /**
+     * Whether the content has more available data to page in
+     *
+     * @function hasMorePages
+     * @returns  {boolean} - True if more content pages are available
+     */
+    hasMorePages: function() {
+        return this.get( 'content.length' ) < this.get( 'totalCount' );
```

Comment:
What do you think of changing "content" to "records" or something other than "content" to avoid this (potential) confusion in the future? My concern is that with "content" meaning something specific in Ember and Ember moving to use "model" over "content" confusion will arise.